setup.cfg

```diff
@@ -28,11 +28,12 @@ copyright-author=Inmanta
 select = E,F,W,C,BLK,I
 
 [isort]
+profile=black
 multi_line_output=3
 include_trailing_comma=True
 force_grid_wrap=0
 use_parentheses=True
-line_length=128
+#line_length=128
```

Comment:
Because without it, I am stuck in a formatting loop between black and isort.  Black seems to ignore the `max-line-length` setting in imports, I have no idea why.